schemas/rules/operators/value/count.json

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://eq.ons.gov.uk/rules/operators/value/count.json",
+  "description": "Returns the number of objects in an array. Accepts a single arguement which is the array to get the number of objects from",
```

Comment:
Minor spelling mistake with "arguement". Would probably be better using the term count rather than number to align with the operator name. I would shorten it slightly, but up to you
```
Returns the count of items in an array. Accepts a single argument which is the array
```